x-pack/plugins/endpoint/public/embeddables/resolver/store/reducer.ts

```diff
@@ -10,12 +10,18 @@ import { dataReducer } from './data/reducer';
 import { ResolverState, ResolverAction, ResolverUIState } from '../types';
 
 const uiReducer: Reducer<ResolverUIState, ResolverAction> = (
-  uiState = { activeDescendentId: null },
```

Comment:
ℹ️ Fun fact: "descendant" and "descendent" (at least in en-US) have slightly different meanings. "descendant" with a "a" is defined as a noun meaning `a person, plant, or animal that is descended from a particular ancestor` and "descendent" with an  adjective "e" meaning `descending from an ancestor`. Given that many other ARIA attributes are also adjectives ("required","relevant") and a document element is difficult to classify as a "person, plant or animal" I would have suggested "descendent" as the better term, but 🤷‍♂